Formula/signal-cli.rb

```diff
@@ -66,14 +66,23 @@ def install
       # rm originally-embedded libzkgroup library
       system "zip", "-d", zkgroup_jar, "libzkgroup.so"
 
+      # https://github.com/Homebrew/homebrew-core/pull/83322#issuecomment-918945146
+      # this fix is needed until signal-cli updates to zkgroup v0.7.3
+      inreplace "rust-toolchain", "1.41.1", "nightly" if Hardware::CPU.arm?
```

Comment:
Can we specify a date on `nightly` so that we get the same Rust compiler across different source builds?

Reply:
Sure!